### ISSUE TITLE
Some type of entry search

### ISSUE THREAD

**Issue by Banjerr** [Banjerr/Grateful-ness#8]
A simple fuzzy search to look for stuff in the title/body

**Comment by ktsuench** [comment]
I'd like to try this out :)

**Comment by Banjerr** [comment]
Awesome! I haven’t gotten any PR guidelines yet, but if you could just make a feature branch from “master” and then make a PR that would be great 🙏

**Comment by petercr** [comment]
Hey @ktsuench I didn't know if you had made any progress on this?
Also I think we would have to build out a UI for the search, right?

Has anyone put any thought into that 🤔

**Comment by Banjerr** [comment]
I have not seen any progress on this so far. We would certainly need a simple UI layout for the search/results; but this could be super minimal as styling/look hasn’t really been thought out yet.

**Comment by ktsuench** [comment]
Nope, I've been busy with some other things but yes looks like we'll need a UI. I was just thinking of implementing the logic.
